nginx/nginx.go

```diff
@@ -488,9 +509,6 @@ func (n *nginxUpdater) Health() error {
 	if !n.running.Get() {
 		return errors.New("nginx is not running")
 	}
-	if !n.initialUpdateApplied.Get() {
-		return errors.New("waiting for initial update")
-	}
```

Comment:
Makes sense, that way we know whether Nginx wasn't yet started or whether it died.